### ISSUE TITLE
SkeletonView creates a giant black view over entire collection view which won't go away

### ISSUE THREAD

**Issue by bryan1anderson** [Juanpe/SkeletonView#173]
### What did you do?
Implemented:
```
    func collectionSkeletonView(_ skeletonView: UICollectionView, cellIdentifierForItemAt indexPath: IndexPath) -> ReusableCellIdentifier {
        return "AreaUserSelectionCell"
    }
```

called `self.collectionView.showGradientSkeleton()` when the view appears
Called `self.collectionView.hideSkeleton(reloadDataAfter: true)` when the data loaded a few seconds later

### What did you expect to happen?
Show a few cells with skeletons while the data loaded, and then remove those
### What happened instead?

A gigantic rectangle nearly the size of the entire collection appeared and won't go away even after I call `self.collectionView.hideSkeleton(reloadDataAfter: true)`

### SkeletonView Environment

**SkeletonView version:*1.7*
**Xcode version:* 10.2.1 (10E1001)*
**Swift version:*5*

**Comment by bryan1anderson** [comment]
![Image](https://user-images.githubusercontent.com/7976521/62881637-73591600-bced-11e9-9412-e7385526a2a6.gif)

**Comment by bryan1anderson** [comment]
![Screen Shot 2019-08-13 at 7 58 57 AM](https://user-images.githubusercontent.com/7976521/62947714-40238f00-bda0-11e9-8b9c-179216acf390.png)
Further investigation. I made the CollectionView to have a blue background. So you can clearly see that for some reason, Skeleton view is adding a gigantic gray view to the whole collection view, while you can still clearly see that the individual cells are correctly layered

**Comment by Juanpe** [comment]
Hi @bryan1anderson, I think your problem was solved in version 1.3. 

Collection views are special because you need to prepare the collection before to show the skeleton. 
You can check the info in this [PR](https://github.com/Juanpe/SkeletonView/pull/42)

Let me know if your problem is solved with this code.

Thanks
